projects/ad4630_fmc/zed/README.md

```diff
@@ -37,81 +37,96 @@ The overwritable parameters from the environment are:
 ```
 
 Corresponding device trees:
 
 - [zynq-zed-adv7511-ad4030-24.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-ad4030-24.dts)
 - [zynq-zed-adv7511-ad4032-24.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-ad4032-24.dts)
-- [zynq-zed-adv7511-ad4630-16.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-ad4630-16.dts)
+- [zynq-zed-adv7511-adaq4216.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4216.dts)
+- [zynq-zed-adv7511-adaq4220.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4220.dts)
+- [zynq-zed-adv7511-adaq4224-24.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4224-24.dts)
 
-#### Clock mode 0, MOSI lines 4, Capture zone 2, SDR
+#### Clock mode SPI, MISO lines 4, Capture zone 2, SDR
 
 ```
-make CLK_MODE=0 NUM_OF_SDI=4 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=0 NUM_OF_SDI=4
 ```
 
-Corresponding device tree: [zynq-zed-adv7511-adaq4224-24_cm0_sdi4_cz2.dts](https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zed-adv7511-adaq4224-24_cm0_sdi4_cz2.dts)
-
-#### Clock mode 0, MOSI lines 8, Capture zone 2, SDR
+#### Clock mode SPI, MISO lines 8, Capture zone 2, SDR
 
 ```
-make CLK_MODE=0 NUM_OF_SDI=8 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=0 NUM_OF_SDI=8
 ```
 
-#### Clock mode 1, MOSI lines 2, Capture zone 2, SDR
+#### Clock mode ECHO, MISO lines 2, Capture zone 2, SDR
 
 ```
-make CLK_MODE=1 NUM_OF_SDI=2 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=1 NUM_OF_SDI=2 CAPTURE_ZONE=2
 ```
 
-#### Clock mode 1, MOSI lines 4, Capture zone 2, SDR
+#### Clock mode ECHO, MISO lines 4, Capture zone 2, SDR
 
 ```
-make CLK_MODE=1 NUM_OF_SDI=4 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=1 NUM_OF_SDI=4 CAPTURE_ZONE=2
 ```
 
-#### Clock mode 1, MOSI lines 8, Capture zone 2, SDR
+#### Clock mode ECHO, MISO lines 8, Capture zone 2, SDR
 
 ```
-make CLK_MODE=1 NUM_OF_SDI=8 CAPTURE_ZONE=2 DDR_EN=0
+make CLK_MODE=1 NUM_OF_SDI=8 CAPTURE_ZONE=2
 ```
 
-#### Clock mode 1, MOSI lines 2, Capture zone 2, DDR
+#### Clock mode ECHO, MISO lines 2, Capture zone 2, DDR
 
 ```
 make CLK_MODE=1 NUM_OF_SDI=2 CAPTURE_ZONE=2 DDR_EN=1
 ```
 
-#### Clock mode 1, MOSI lines 4, Capture zone 2, DDR
+#### Clock mode ECHO, MISO lines 4, Capture zone 2, DDR
 
 ```
 make CLK_MODE=1 NUM_OF_SDI=4 CAPTURE_ZONE=2 DDR_EN=1
 ```
 
-#### Clock mode 1, MOSI lines 8, Capture zone 2, DDR
+#### Clock mode ECHO, MISO lines 8, Capture zone 2, DDR
 
 ```
 make CLK_MODE=1 NUM_OF_SDI=8 CAPTURE_ZONE=2 DDR_EN=1
-```
\ No newline at end of file
+```
+
+#### Clock mode SPI, MISO lines 1, Capture zone 1, SDR (ad4030)
+
+```
+make CLK_MODE=0 NUM_OF_SDI=1 CAPTURE_ZONE=1 NO_REORDER=1
+```
+
+#### Clock mode SPI, MISO lines 2, Capture zone 1, SDR (ad4630)
```

Comment:
If you specified here the devices (**AD4630** and **AD4030**), shouldn't they be specified then even in the previous examples as well?

Reply:
No, Other build configs works for all devices AD4030 AD4630 ADAQ42xx and others

Reply:
Done!